webapp/views/launch/script.js

```diff
@@ -530,28 +533,29 @@ function saveDrawing(res) {
 }
 
 function getOutput() {
-  get('/output', {}, function (res) {
-    const baseOption = "<option selected value=''> - </option>"
-    const options = res.list.reduce((str, file) => str + `<option value="${file}">${file}</option>`, '')
-    $('#results-select').html(baseOption + options)
-  })
+  get('/output', {}, (res) => new Promise(() => {
+    const baseOption = "<option selected value=''> - </option>";
+    const options = res.list.reduce((str, file) => str + `<option value="${file}">${file}</option>`, '');
+    $('#results-select').html(baseOption + options);
+  }));
```

Comment:
Shouldn't this Promise have a `resolve` as well?

Reply:
I works this way, too, because it resolves by default, but I agree, resolve should be called for clarity.